Source/Flow/Public/Interfaces/FlowDataPinValueSupplierInterface.h

```diff
@@ -0,0 +1,88 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "Types/FlowDataPinResults.h"
+
+#include "UObject/Interface.h"
+
+#include "FlowDataPinValueSupplierInterface.generated.h"
+
+// Interface to define a Flow Data Pin value supplier.  This is generally a UFlowNode subclass, 
+// but we may support external suppliers that are not flow nodes in the future 
+// (eg, for supplying configuration values for the root graph)
+UINTERFACE(MinimalAPI, Blueprintable, DisplayName = "Flow Data Pin Value Supplier Interface")
+class UFlowDataPinValueSupplierInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowDataPinValueSupplierInterface
```

Comment:
This is an interface so that we can theoretically have non-flownode uobjects serve as suppliers (like a component).